docs/src/pages/css-in-js/basics/basics.md

```diff
@@ -48,6 +48,7 @@ install();
 
 It is **recommended** to place the above code in a separate file (e.g. `bootstrap.js`) and to import it in your application's entry point (e.g. `index.js`).
 This ensures that the installation is executed before anything else, because ECMAScript imports are hoisted to the top of the module. If the installation step is not performed correctly the resulting build could have conflicting class names.
+Also remember wrapping your Component Tree with the `ThemeProvider` that is provided by `@material-ui/styles` as it is not the same as the one from `@material-ui/core`.
```

Comment:
ThemeProvider from @material-ui/core is on it's way out. Maybe we could say that instead?

Reply:
I changed it to your proposition. Does that mean I can already remove the ThemeProvider from `@material-ui/core`, or do I still need it until 4.0?

Reply:
If you are not using any custom theme, you can remove it. If you are, you can switch the usage.